src/components/Item.js

```diff
@@ -1,10 +1,69 @@
 import React from "react";
+import { useAppReducer } from "../AppContext";
+import styles from "./Item.module.scss";
 
 // Individial todo item
 function Item({ item }) {
+  const dispatch = useAppReducer();
+  let text = item.text;
+  let paused = item.status === "paused";
+  let completed = item.status === "completed";
+
+  function deleteItem() {
+    dispatch({ type: "DELETE_ITEM", item });
+  }
+
+  function pauseItem() {
+    const pausedItem = { ...item, status: "paused" };
```

Comment:
`...item` is unnecessary

Reply:
double merging of the object here and in the reducer, I think it will be more pronounced in this way:
```
function pauseItem() {
    const partialData = { status: "paused" };
    dispatch({ type: "UPDATE_ITEM", data: partialData });
  }
```